_inc/client/components/dev-card/index.jsx

```diff
@@ -8,19 +8,110 @@ import classNames from 'classnames';
 /**
  * Internal dependencies
  */
-import { isDevVersion as _isDevVersion } from 'state/initial-state';
-import { switchPlanPreview } from 'state/dev-version';
+import {
+	isDevVersion as _isDevVersion,
+	userCanViewStats,
+	userIsMaster,
+	userCanDisconnectSite,
+	userCanEditPosts
+} from 'state/initial-state';
 import { getSitePlan } from 'state/site';
-import { canDisplayDevCard, disableDevCard } from 'state/dev-version';
+import { isCurrentUserLinked } from 'state/connection';
+import {
+	switchPlanPreview,
+	canDisplayDevCard,
+	disableDevCard,
+	switchUserPermission
+} from 'state/dev-version';
 import Card from 'components/card';
 
 export const DevCard = React.createClass( {
 	displayName: 'DevCard',
 
-	onChange( event ) {
+	onPlanChange( event ) {
 		this.props.switchPlanPreview( event.target.value );
 	},
 
+	onPermissionsChange( event ) {
+		this.props.switchUserPermissions( event.target.value );
+	},
+
+	maybeShowStatsToggle() {
+		if ( ! this.props.isAdmin ) {
+			return (
+				<div>
+					<hr/>
+					<ul>
+						<li>
+							<label>
+								<input
+									type='radio'
```

Comment:
Can you quickly update these `'` to be `"`? It introduced 16 new warnings to ESLint.